README.md

```diff
@@ -3,3 +3,17 @@
 # `lnschema_core`
 
 `lnschema_core` is documented & tested within `lamindb`: [github.com/laminlabs/lamindb](https://github.com/laminlabs/lamindb).
+
+## Developers' Note
+
+This package implements models but leaves concrete implementations of non-ORM methods (such as `Artifact.cache`) empty. The methods are implemented in `lamindb` and then the model classes are monkey-patched there as well.
+
+### Why monkey pathing?
+
+We want to avoid implementing anything non-ORM related in this package and just have models here, so that concrete implementations are all in `lamindb`. This provides better code structure and clarity compared to having imports from `lamindb` needed if the implementations are done here.
```

Comment:
Hmmm, the first sentence doesn't provide a reason. The second seems subjective. We need to be clearer.

Reply:
>One reason is auto-connect and the difficulties of setting up Django.

I don't think it is the reason. Because we actually can import `lamindb` inside methods and do implemetations in `lnschema_core`. And we had this some time ago. But the code structure and clarity was much worse.

Reply:
Ok, but this means we _could_ eliminate lnschema-core and just have everything in lamindb and work with lamindb itself as the most basic Django app.

There'll be some weirdness about lamindb-setup then depending on lamindb instead of lnschema-core and we should likely then also move lamindb-setup into lamindb; but it seems possible. 🤔 

Of course, all of this would be considerable work in particular for creating the CI architecture of a mono-repo, but also that'd be possible.

Finally it'd be great to integrate lamin-cli and I don't yet see how that's possible without introducing latency but I'm pretty sure we could also overcome this. So, a `lamindb` mono-repo vision is conceivable. :D 

All of this would need to be done during a Christmas break or so because of the disruptiveness of the update process. But once it's done we'd never want to change it again, I imagine.

Reply:
> Ok, but this means we could eliminate lnschema-core and just have everything in lamindb and work with lamindb itself as the most basic Django app.
Maybe it is possible, but requires some additional pretty complicated checks on `lamindb` import. But i don't really see how it follows from my comment actually. What i was talking about is importing things from `lamindb` inside `lnchema_core`, we had this, this is a pretty uncomplicated settings because at the point of import inside a methods everything was already setup. I am not sure it is easily adaptable to the single repo variant.